pgjdbc/src/main/java/org/postgresql/jdbc/PgDatabaseMetaData.java

```diff
@@ -2340,47 +2340,33 @@ public ResultSet getPrimaryKeys(@Nullable String catalog, @Nullable String schem
       throws SQLException {
     String sql;
 
-    // Version 11 added "include columns" in index hence we need to filter only the key attributes
-    // when returning primary keys.
-    String keyCountColumn = connection.haveMinimumServerVersion(ServerVersion.v11) ? "i.indnkeyatts" : "i.indnatts";
-
-    sql = "SELECT current_database() AS TABLE_CAT, n.nspname AS TABLE_SCHEM, "
```

Comment:
I think you need to rebase. This code where we are selecting `current_database() as TABLE_CAT` is in HEAD

Reply:
This has not changed since I forked this branch, but in the version I am proposing I have `table_catalog=current_database()` in the `WHERE` clause so I can select `table_catalog` as `TABLE_CAT`

Reply:
One more thing to note about this line, in the old version there was an outer query with quoted upper case identifiers and this inner query with unquoted identifiers. In the new version there are no subqueries, so I am returning quoted upper-case identifiers as was the case in the original outer query.